sum-of-multiples/example.js

```diff
@@ -0,0 +1,18 @@
+const Sum = (factors = [3, 5]) => {
+  let self = this instanceof Sum ? this : Object.getPrototypeOf(Sum);
```

Comment:
It's a strange statement. I haven't seem anything like this before. I'm ok with this, but just for the purpose of learning. I would have written the `example.js` in a different way. But I think it's ok to be merged.